scripts/countdown_erc721/README.md

```diff
@@ -120,3 +120,27 @@ This is a straightforward script that invokes the contract `purchase` function u
 ```sh
 ts-node ./scripts/countdown_erc721/4-purchase.ts 
 ```
+
+## Script 5: Set Metadata 
+
+This is a straightforward script that invokes the contract `setMetadataParams` function using hardcoded values as parameters. This script also allows the execution to be made using a Safe wallet.
+
+PS: Please ensure that you execute the script using a private key belonging to one of the Safe address owners if you're running the script with the Safe wallet.
```

Comment:
@BravoNatalie i like this call out. Is there a way to validate this before its submitted? 

What happens if i used an unapproved wallet? What kind of error do i get?

Reply:
The SDK does not allow it.

![image](https://github.com/holographxyz/holograph-protocol/assets/37748828/5ae6b1ef-1254-4b0a-b39a-ec932ecd5a30)